src/modules/reading-room/const/index.tsx

```diff
@@ -26,7 +26,7 @@ export const READING_ROOM_ITEM_COUNT = 100;
 export const READING_ROOM_QUERY_PARAM_INIT = {
 	format: ReadingRoomMediaType.All,
 	orderProp: ReadingRoomSort.Relevance,
-	page: 1,
+	page: 0,
```

Comment:
All page params in the app start counting from 1.
The backend also uses this rule.

Should this one be different for some reason?

Reply:
Pagination was non-functional because it's 0-indexed, reverted and adjusted there